MarkSkalskyi/README.md

```diff
@@ -0,0 +1,42 @@
+# Getting started with [HDRezka](https://rezka.ag)
+
+HDRezka is a website that allows you to watch your favorite movies and series. If you need a guide for signing in, you're in the right place.
+
+**Movies? HDRezka!**
+
+## Step 1: Registration
+
+1. Firstly, visit the HDRezka website at [HDRezka](https://rezka.ag).
```

Comment:
This is step numbered 1. You don't need to use such words as firstly, secondly, etc.